docs/schema_plugins.md

```diff
@@ -1 +1,234 @@
+    """
+
+    name = Quantity(
+        type=str,
+        shape=[],
+        description="""
+        Specifies the name of the author who generated the h5md file.
+        """,
+    )
+
+    email = Quantity(
+        type=str,
+        shape=[],
+        description="""
+        Author's email.
+        """,
+    )
+```
+
+Again, we need to add this to our custom `Simulation()` class:
+
+ ```python
+class Simulation(nomad_simulations.schema_packages.general.Simulation):
+
+    x_h5md_creator = SubSection(
+        sub_section=nomad_simulations.schema_packages.general.Program.m_def
+    )
+
+    x_h5md_author = SubSection(sub_section=Author.m_def)
+```
+
+And, in our parser:
+
+```python
+from nomad_simulations.schema_packages/<parser_name>_schema.py import Simulation, Author
+
+simulation = Simulation()
+simulation.x_h5md_author = Author(
+    name=group_h5md_dict.get('h5md_author_name'),
+    email=group_h5md_dict.get('h5md_author_email'),
+)
+
+```
+
+Finally, we need to store the h5md-nomad schema version. We could simply add a string quantity to `Simulation()`, however since the h5md schema uses semantic versioning, we could store this as a list of 3 integers to ensure that the user provides the relevant information for this quantity. In `schema_packages/<parser_name>_schema.py`:
+
+
+ ```python
+class Simulation(nomad_simulations.schema_packages.general.Simulation):
+
+    x_h5md_creator = SubSection(
+        sub_section=nomad_simulations.schema_packages.general.Program.m_def
+    )
+
+    x_h5md_author = SubSection(sub_section=Author.m_def)
+
+    x_h5md_version = Quantity(
+        type=np.dtype(np.int32),
+        shape=[3],
+        description="""
+        Specifies the version of the h5md schema being followed.
+        """,
+    )
+```
+
+And in the parser:
+
+```python
+from nomad_simulations.schema_packages/<parser_name>_schema.py import Simulation, Author
+
+simulation = Simulation()
+simulation.x_h5md_version = group_h5md_dict.get('h5md_version')
+```
+
+
+Now, imagine that your simulation outputs some observable that is not yet defined within `nomad-simulations`. We can easily extend the schema following the approach from above (in `schema_packages/<parser_name>_schema.py`):
+
+```python
+class Stress(nomad_simulations.schema_packages.physical_property.PhysicalProperty):
+    """
+    Here you should give a detailed description of this property.
+    """
+
+    value = Quantity(
+        type=np.dtype(np.float64),
+        unit='newton',
+    )
+
+class Outputs(nomad_simulations.schema_packages.outputs.Outputs):
+
+
+    stress = SubSection(
+        sub_section=Stress.m_def,
+        repeats=True,
+    )
+```
+
+And in the parser:
+
+
+```python
+from nomad_simulations.schema_packages/<parser_name>_schema.py import Outputs, Stress
+from nomad.units import ureg
+
+output = Outputs()
+simulation.outputs.append(output)
+output.stress.append(
+  Stress(value=<value_from_file_parsing> * ureg.newton)
+)
+```
```

Comment:
Same sentiment about the choice: this is again an example of semantic extension. Maybe you could add 1 more showing when one should refrain overwriting, or how to interact with quantities that have links/IRIs.

At this stage in the tutorial, I'd more so look for examples of _enhanced normalization_ and maybe _polymorphism_.